Code/Tools/SceneAPI/SceneCore/DataTypes/GraphData/IMeshVertexTangentData.h

```diff
@@ -16,42 +16,36 @@ namespace AZ
     class Vector4;
 }
 
-namespace AZ
+namespace AZ::SceneAPI::DataTypes
 {
-    namespace SceneAPI
+    enum class TangentGenerationMethod
+    {
+        FromSourceScene = 0,
+        MikkT = 1
+    };
+
+    enum class MikkTSpaceMethod
```

Comment:
What's the difference between TSpace and TSpaceBasic? I think perhaps we should name it something like TSpaceTrue and TSpaceBasic (Maybe TSpaceFast)?
Since this is done at asset time, what's the benefit of using the TSpaceBasic method?

Reply:
No standard term but also a MikkT specific setting. MikkT provides two different methods to generate the tangents and they are called like that. Happy to rename them if there are any recommendations.